src/AltinnCore/Common/Services/Implementation/SourceControlSI.cs

```diff
@@ -586,12 +586,12 @@ public void CheckoutLatestCommitForSpecificFile(string owner, string repository,
         /// <summary>
         /// Stages a specific file changed in working repository.
         /// </summary>
-        /// <param name="owner">The owner of the repository.</param>
+        /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
         /// <param name="repository">The name of the repository.</param>
         /// <param name="fileName">the entire file path with filen name</param>
-        public void StageChange(string owner, string repository, string fileName)
+        public void StageChange(string org, string repository, string fileName)
```

Comment:
i dont see that this call is updated in react

Reply:
Same here, it doesn't look like it is in use in react.